### PR TITLE
New-Reviews-and-Ratings-API

### PR DESCRIPTION
**Pull request by karenkrieger**
#### Types of changes
- [X] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

### CODE REVIEW

**Review by Arturo777 — DISMISSED** [review]
LGTM

File: VTEX - Reviews and Ratings API.json [hunk 1]
```diff
@@ -0,0 +1,1171 @@
...
+                "properties": {
+                    "productId": {
+                        "type": "string"
+                    },
+                    "rating": {
+                        "type": "integer",
+                        "format": "int32"
+                    },
+                    "title": {
+                        "type": "string"
```
Comment (PedroAntunesCosta) [inline VTEX - Reviews and Ratings API.json:1088]: Adicionar `description`.

File: VTEX - Reviews and Ratings API.json [hunk 2]
```diff
@@ -0,0 +1,1171 @@
...
+                    },
+                    "text": {
+                        "type": "string"
```
Comment (PedroAntunesCosta) [inline VTEX - Reviews and Ratings API.json:1091]: Adicionar `description`.

File: VTEX - Reviews and Ratings API.json [hunk 3]
```diff
@@ -0,0 +1,1171 @@
...
+                    },
+                    "reviewerName": {
+                        "type": "string"
```
Comment (PedroAntunesCosta) [inline VTEX - Reviews and Ratings API.json:1094]: Adicionar `description`.

**Review by PedroAntunesCosta — APPROVED** [review]
LGTM

**Pull request merged**